translations/en/events/taravangians-treachery.md

```diff
@@ -0,0 +1,14 @@
+# Taravangian's treachery
+Taravangian meets with the inner circle of the Diagram and the group disbands.

```

Comment:
This summary duplicated with `diagram-disbands.md` @jofwu

Reply:
@lianzhao I've updated this one! Thanks.